### ISSUE TITLE
elysia Js

### ISSUE THREAD

**Issue by VigilioYonatan** [serhiisol/node-decorators#201]
Node decorators thought about working with Elysia JS?

**Comment by serhiisol** [comment]
@VigilioYonatan hi, do you mean to support in the newest version of decorators (i.e. `@decorators/server`)?

**Comment by VigilioYonatan** [comment]
yes. elysia js is fast and lightweight framework. it look like express

**Comment by angelxmoreno** [comment]
I find this interesting, especially since I am already using `bun` and `@decorators/express`. 

Did anything coms about this @serhiisol @VigilioYonatan ?
